roofit/roofit/src/RooExpPoly.cxx

```diff
@@ -0,0 +1,311 @@
+/*****************************************************************************
+ * Project: RooFit                                                           *
+ * Package: RooFitModels                                                     *
+ * @(#)root/roofit:$Id$
+ * Authors:                                                                  *
+ *   WV, Wouter Verkerke, UC Santa Barbara, verkerke@slac.stanford.edu       *
+ *   DK, David Kirkby,    UC Irvine,         dkirkby@uci.edu                 *
+ *                                                                           *
+ * Copyright (c) 2000-2005, Regents of the University of California          *
+ *                          and Stanford University. All rights reserved.    *
+ *                                                                           *
+ * Redistribution and use in source and binary forms,                        *
+ * with or without modification, are permitted according to the terms        *
+ * listed in LICENSE (http://roofit.sourceforge.net/license.txt)             *
+ *****************************************************************************/
+
+/** \class RooExpPoly
+    \ingroup Roofit
+
+RooExpPoly implements a polynomial p.d.f of the form \f[ f(x) =
+\mathcal{N} \cdot \exp( \sum_{i} a_{i} * x^{i} ) \f] \f$ \mathcal{N}
+\f$ is a normalisation constant that is automatically calculated when
+the function is used in computations.
+
+The sum can be truncated at the low end. See the main constructor
+RooExpPoly::RooExpPoly(const char*, const char*, RooAbsReal&, const RooArgList&, Int_t)
+
+\image html RooExpPoly.png
+
+**/
+
+#include <cmath>
+#include <sstream>
+#include <cassert>
+
+#include "TMath.h"
+
+#include "RooExpPoly.h"
+#include "RooAbsReal.h"
+#include "RooArgList.h"
+#include "RooMsgService.h"
+
+#include "TError.h"
```

Comment:
This might fix the compile errors. (Not sure if I got it all.)
```suggestion
#include "RooExpPoly.h"

#include "RooAbsReal.h"
#include "RooArgList.h"
#include "RooMsgService.h"
#include "RooRealVar.h"

#include "TMath.h"
#include "TError.h"

#include <cmath>
#include <sstream>
#include <cassert>
#include <complex>
```